simple_ldap_user/simple_ldap_user.module

```diff
@@ -480,8 +480,17 @@ function simple_ldap_user_sync_user_to_drupal($drupal_user) {
   // Load the LDAP user, force a cache reset.
   $ldap_user = SimpleLdapUser::singleton($drupal_user->name, TRUE);
 
-  // Nothing to sync.
-  if (!$ldap_user->exists || (isset($drupal_user->uid) && $drupal_user->uid == 1)) {
+  // Nothing to sync. Check if user is an admin
+  if(isset($drupal_user->uid) && $drupal_user->uid == 1) {
+    return;
+  } //For other users
+  elseif (!$ldap_user->exists) {
+    $result = db_update('users')
```

Comment:
This update statement will work for the bug report and patch to the upstream code.

It isn't suitable to be pulled into `uls-install` by itself because of:
https://github.com/ulsdevteam/simple_ldap/commit/841319964afb2e1b67202135413b50b7eb6fc8e2#diff-490f9ed71aad5cbb1c402a18c4bc583aR211
In some cases, we may want to allow local Drupal users to be maintained, even without a connection to the LDAP user.

We'll need another commit in the [readonly-ldap-users-roles](https://github.com/ulsdevteam/simple_ldap/commits/readonly-ldap-users-roles/simple_ldap_user) branch which also accounts for the `SimpleLdapUser::allowOrphans()` condition.